auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -347,9 +346,7 @@ public Void then(@NonNull Task<GoogleApiClient> task) throws Exception {
                 });
 
         // Facebook sign out
-        if (FacebookSdk.isInitialized()) {
```

Comment:
Did something change the Facebook SDK that we no longer need to deal with initialization?